learn/00_introduction.md

```diff
@@ -1,36 +1,55 @@
 # Introduction
 
-Week Goal: Get excited about the programme and what you are going to learnt over the next 12 weeks.
```

Comment:
I think it's a shame to lose the weekly goal. Think it's good to have something to refer back to at the end of the week or for retrospectives, to see if we've achieved what we've set out to.